content/v2.0-preview/get-started/get-started-with-managed-resources.md

```diff
@@ -152,33 +157,29 @@ spec:
 EOF
 ```
 
-This attaches the AWS credentials, saved as a Kubernetes secret, as a 
+This attaches the AWS credentials, saved as a Kubernetes secret, as a
 {{< hover label="providerconfig" line="9">}}secretRef{{</ hover>}}.
 
-The 
-{{< hover label="providerconfig" line="11">}}spec.credentials.secretRef.name{{< /hover >}} 
-value is the name of the Kubernetes secret containing the AWS credentials in the 
-{{< hover label="providerconfig" line="10">}}spec.credentials.secretRef.namespace{{< /hover >}}.
+## Create a namespace
+Before we can create our namespaced S3 bucket managed resource, we must create a
+namespace for it.
```

Comment:
Can we omit this and just use the `default` namespace? That's what I'm doing in the Composition docs.

Reply:
yes that's definitely a better approach, no need to create a new namespace 🤓